pkg/node/fsm/fsm_common.go

```diff
@@ -119,7 +120,12 @@ func tryBroadcastTransaction(
 			}
 		}()
 	}
-	if _, err = t.Validate(baseInfo.scheme); err != nil {
+	lightNodeActivated, err := baseInfo.storage.IsActivated(int16(settings.LightNode))
```

Comment:
The same as above

Reply:
Here it's also OK, transaction gets into UTX from  here. Think of it as a pre validation. If a transaction gets invalidated because of activation of feature 22 what is the probability for it to became valid? I thinks it's zero.

Reply:
Okay